dpnp/tests/third_party/cupy/statistics_tests/test_histogram.py

```diff
@@ -474,7 +473,7 @@ def test_digitize_nd_bins(self):
 )
 class TestHistogramdd:
     @testing.for_all_dtypes(no_bool=True, no_complex=True)
-    @testing.numpy_cupy_allclose(atol=1e-7, rtol=1e-7)
+    @testing.numpy_cupy_allclose(atol=1e-3, rtol=1e-3, type_check=False)
```

Comment:
Is it expected to have different dtypes, considering we have `asarray(.., dtype=expected_dtype,..) at the end of `histogramdd` function?

Reply:
Replaced with `type_check=has_support_aspect64()`

Reply:
The tests failed due to result type mismatch when running with numpy 2.1.
It seems numpy uses different type promotion in the latest version. Is there something we should align with?
Otherwise we have to keep `type_check=False`.